.gitignore

```diff
@@ -15,3 +15,4 @@ result
 
 # Site build
 /dist/
+.direnv
```

Comment:
Add to your global gitignore instead

Reply:
![CleanShot 2025-04-23 at 19 55 38@2x](https://github.com/user-attachments/assets/0484f845-f2a2-4319-9840-96b2d6bc8ae9)

:P

(ok)

Reply:
Dropped this part of the PR